app/controllers/posts_controller.rb

```diff
@@ -1,31 +1,52 @@
 class PostsController < ApplicationController
+  before_action :find_post, only: [:destroy]
+
```

Comment:
- [ ] Great job adding the `destroy` method to the post and comment controllers, however kindly make sure to add the `load_and_authorize_resource` statement to the PostController

```suggestion
class PostsController < ApplicationController
  load_and_authorize_resource
  before_action :find_post, only: [:destroy]

```